pkg/api/validation/validation_test.go

```diff
@@ -655,6 +655,7 @@ func TestValidateVolumes(t *testing.T) {
 		{Name: "rbd", VolumeSource: api.VolumeSource{RBD: &api.RBDVolumeSource{CephMonitors: []string{"foo"}, RBDImage: "bar", FSType: "ext4"}}},
 		{Name: "cinder", VolumeSource: api.VolumeSource{Cinder: &api.CinderVolumeSource{VolumeID: "29ea5088-4f60-4757-962e-dba678767887", FSType: "ext4", ReadOnly: false}}},
 		{Name: "cephfs", VolumeSource: api.VolumeSource{CephFS: &api.CephFSVolumeSource{Monitors: []string{"foo"}}}},
+		{Name: "quobyte", VolumeSource: api.VolumeSource{Quobyte: &api.QuobyteVolumeSource{Registry: "host1:7861", Volume: "volume", ReadOnly: false}}},
```

Comment:
this is good, can you also add negative tests too, e.g. empty registry and/or volume